vulture.py

```diff
@@ -394,6 +395,10 @@ def csv(option, _, value, parser):
 
 def main():
     options, args = parse_args()
+    whitelist_path = os.path.abspath(stdlib.__file__)
+    if whitelist_path.endswith('.pyc'):
+        whitelist_path = whitelist_path[:-1]
+    args.append(whitelist_path)
```

Comment:
This solution only works when calling vulture from the command line, not as a library. You can add the following code into scavenge  

```
modules = self._get_modules(paths)
modules.append(_get_stdlib_whitelist_file())
```
and somewhere at the top of the script

```
def _get_stdlib_whitelist_file():
    script = os.path.abspath(__file__)
    whitelist_dir = os.path.dirname(script)
    return os.path.join(whitelist_dir, 'stdlib.py')
```

As a side effect, users can now disable default whitelist files by using `--exclude whitelists` since "whitelists" is part of the path name.